frontend/server/src/Controllers/Problem.php

```diff
@@ -4824,11 +4825,18 @@ public static function getProblemListForSmarty(
     public static function getProblemCollectionDetailsForSmarty(
         \OmegaUp\Request $r
     ): array {
+        $tags = [];
+        $allTags = self::getAllTagsFromCache();
+
+        foreach ($allTags as $tag) {
+            $tags[] = ['name' => $tag->name];
```

Comment:
Para que te deje de marcar el error de psalm puedes omitir todos los tags donde el nombre sea `null`, algo como esto:
```suggestion
            if (is_null($tag->name)) {
                continue;
            }
            $tags[] = ['name' => $tag->name];
```